__TESTS__/unit/values/gravity/AutoGravity.test.ts

```diff
@@ -0,0 +1,24 @@
+import {Gravity} from "../../../../src/values/gravity/Gravity";
+import {AutoFocus} from "../../../../src/values/gravity/qualifiers/autoFocus/AutoFocus";
+import {FocusOn} from "../../../../src/values/gravity/qualifiers/focusOn/FocusOn";
+
+describe('Gravity Qualifier', () => {
```

Comment:
I didn't see a test/usage example for g_auto with weights and `avoid`, I'd add a couple of tests here (unless this is tested elsewhere and I missed it?)